web/server/codechecker_server/session_manager.py

```diff
@@ -495,6 +480,10 @@ def __try_personal_access_token(self, auth_string):
         if not personal_access_token:
             return False
 
+        if (personal_access_token.expiration - datetime.now()) \
```

Comment:
It is a question of coding style, but can these date fields be simply compared with the `<` operator? It might be more intuitive than checking the result of a subtraction, but you may chose according to your preference.